pkg/reconciler/knativeserving/ingress/ingress_test.go

```diff
@@ -447,6 +447,7 @@ func TestFilters(t *testing.T) {
 	}
 }
 
+// TODO: This test verifies the number of transformers. It should be rewritten by better test.
```

Comment:
I would elaborate on this in a issue and link here

Reply:
Alright! Opened https://github.com/knative/operator/issues/477